app/code/local/Mowdirect/Emailimporter/Helper/Vendor.php

```diff
@@ -1,7 +1,15 @@
 <?php
 
 class Mowdirect_Emailimporter_Helper_Vendor extends Mage_Core_Helper_Abstract {
-    
+
+    public function get_vendors() {
+        $vendors = Mage::getModel('udropship/vendor')->getCollection()->getData();
+        foreach ($vendors as $key => $vendor){
+           $custom_data = json_decode($vendor['custom_vars_combined'], true);
```

Comment:
@sudarshann was `custom_vars_combined` already part of the model?

Reply:
Yes Wayne this already part of the model. It was kept as json encoded. thats why we need a decoding of it before processing the vendor loop.